modules/csvparser/filterx-func-parse-csv.c

```diff
@@ -71,6 +72,9 @@ _parse_list_argument(FilterXFunctionParseCSV *self, FilterXExpr *list_expr, GLis
   if (!filterx_object_len(list_obj, &size))
     return FALSE;
 
+  if (list_length)
+    *list_length = size;
+
```

Comment:
I don't see why we need to extract the list argument into a GList, we could just as easily use the list directly, couldn't we?

Earlier, extracting the string values from a list was cumbersome, but now with filterx_object_extract_string() this is not any more difficult than iterating a GList with strdupped strings on it.

But like I said with my previous comment, we should probably push this into a separate PR.